src/transaction/transaction_manager.cpp

```diff
@@ -28,9 +28,9 @@ timestamp_t TransactionManager::Commit(TransactionContext *const txn) {
   table_latch_.Lock();
   const timestamp_t start_time = txn->StartTime();
   size_t result UNUSED_ATTRIBUTE = curr_running_txns_.erase(start_time);
-  TERRIER_ASSERT(result == 1, "committed transaction did not exist in global transactions table");
+  TERRIER_ASSERT(result == 1, "Committed transaction did not exist in global transactions table");
```

Comment:
If the compiler doesn't object, the following may be preferable, so there are no unnecessary parts in release mode.  Delete the size_t declaration above to go with this.

TERRIER_ASSERT(curr_running_txns_.erase(start_time)== 1, "Committed transaction did not exist in global transactions table");

Reply:
The compiler won't complain, but we can't put the erase() inside of a TERRIER_ASSERT because we still want the erase() call in Release mode.